setup.py

```diff
@@ -76,6 +78,17 @@ def get_config_schema():
         else:
             default_ldflags = []
 
+    if os.environ.get("CIBUILDWHEEL", False) and sys.platform == "win32":
+        import struct
+
+        default_cxxflags.append("-ID:/a/pyopencl/pyopencl/OpenCL-Headers/install/include")
+        if struct.calcsize("P") == 8:
+            default_ldflags.append("/LIBPATH:C:/Program Files/OpenCL-ICD-Loader/lib")
+            default_ldflags.append("/LIBPATH:C:/Program Files/OpenCL-ICD-Loader/bin")
+        else:
+            default_ldflags.append("/LIBPATH:C:/Program Files (x86)/OpenCL-ICD-Loader/lib")
+            default_ldflags.append("/LIBPATH:C:/Program Files (x86)/OpenCL-ICD-Loader/bin")
```

Comment:
Any chance that this could be done from the build script instead of adding clutter to `setup.py`?

Reply:
I try using  CXXFLAGS and LDFLAGS environment variables but it looks like they are ignored. Because `cibuildwheel` use `python -m build` for building wheel then this needs to be added or add the feature to read values from environment variables. What did You prefer?

Reply:
Any way to inject the `siteconf.py` file generated by `configure.py` into cibuildwheel's build process?

Reply:
There are `CIBW_BEFORE_ALL` and `CIBW_BEFORE_BUILD` steps executed before all builds or before every build.
fi configure.py could be used in batch mode then it could be called.